buildout.cfg

```diff
@@ -17,6 +17,7 @@ parts +=
     node
```

Comment:
@hvelarde can I keep these configurations? helps me when debugging and creating RF tests